src/explore-education-statistics-admin/src/pages/release/edit-release/content/ReleaseContentPage.tsx

```diff
@@ -84,14 +84,14 @@ const getUnresolvedComments = (release: EditableRelease) =>
     ...contentSectionComments(release.keyStatisticsSection),
     ...release.content
       .filter(_ => _.content !== undefined)
-      .reduce<ExtendedComment[]>(
+      .reduce<Comment[]>(
         (allComments, contentSection) => [
           ...allComments,
           ...contentSectionComments(contentSection),
         ],
         [],
       ),
-  ].filter(comment => comment !== undefined && comment.state === 'open');
+  ].filter(comment => comment !== undefined);
```

Comment:
Do we not have the notion of comment state anymore then?

Reply:
No we don't because it's not a requirement according to Laura although we could reintroduce it. State was always Open on all comments that exist.